src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -147,6 +147,20 @@ public long getTargetSSRC()
      */
     public boolean accept(byte[] buf, int off, int len)
     {
+        int targetIndex = bitstreamController.getTargetIndex()
+            , currentIndex = bitstreamController.getCurrentIndex();
+
+        if (targetIndex < 0 && currentIndex > -1)
+        {
+            synchronized (this)
```

Comment:
i'm not sure the synchronized here is helping much, is it?  it's possible `setTargetIndex` is called right before this (when the bitrateController is re-assigned).  from what i understand, we could do it this way and get rid of the need for synchronization:

```
BitstreamController new = new BitstreamController(bitstreamController, -1);
this.bitstreamController = new;
```
does that seem correct?